packages/fast-check/src/arbitrary/_internals/builders/CharacterRangeArbitraryBuilder.ts

```diff
@@ -30,7 +30,10 @@ function percentCharArbUnmapper(value: unknown): string {
 }
 
 /** @internal */
-const percentCharArb = fullUnicode().map(percentCharArbMapper, percentCharArbUnmapper);
+const percentCharArb = string({ unit: 'binary', minLength: 1, maxLength: 1 }).map(
```

Comment:
Regression here!!!

Reply:
Going to `const assertTimeout = 10000` in the buggy test is enough

Reply:
The problem is that `createSlicesForString` is too slow and called at init time while it should not